### ISSUE TITLE
UX Suggestion: Character Creation Progression

### ISSUE THREAD

**Issue by chrisdhanaraj** [wanderers-guide/wanderers-guide-legacy#44]
Hello! I can pick this one up but wanted to put it out for discussion. When I was going through the character creation flow, there were two sticky points that jumped out at me

1. There's no _next_ button as you go through the `ABCDE` flow, you have to click the stepper at the top.
2. The primary action is in the small tile versus the main content tile.

Proposing that we add a next button through the flow, and then move the ancestry/background/etc selection button to the main content tile

**Comment by TimWeidner** [comment]
Agreed, The lack of a "next" button is a very important issue in my book. Sadly all the other character building tools I know fall flat in that regard. Here are some examples of how other tools do it.

The only tool I know that does this is sadly D&DB
![image](https://user-images.githubusercontent.com/55436069/215256709-97e7a478-a11e-4c19-adcb-74e298752802.png)

Nexus at least adds a checkmark when all available options have been selected
![image](https://user-images.githubusercontent.com/55436069/215256971-5a6a0336-9f6c-4192-b45b-4342d6357fce.png)

Pathbuilder.. is just a UX nightmare, where there are no steps and only one screen with far too many modals and I don't think it needs a screenshot.
